<launch>
	<arg name="spin_rate" default="100"/>
	<arg name="max_speed" default="30"/>

	<node name="pharos_remote_vehicle_control_node" pkg="pharos_remote_vehicle_control" type="pharos_remote_vehicle_control_node" output="screen">
		<param name="spin_rate" value="$(arg spin_rate)"/>
		<param name="max_speed" value="$(arg max_speed)"/>

		<!-- <remap from="test_pub" to="test_pub" /> -->
		<!-- <remap from="test_sub" to="test_sub" /> -->
	</node>

	<arg name="role_name" default="ego_vehicle" />
	<arg name="control_loop_rate" default="0.0333" />

	<!-- <node pkg="carla_ackermann_control" type="carla_ackermann_control_node.py" name="carla_ackermann_control_$(arg role_name)" output="screen">
		<param name="role_name" value="$(arg role_name)" />
		<param name="control_loop_rate" value="$(arg control_loop_rate)" />
		<rosparam file="$(find carla_ackermann_control)/config/settings.ros1.yaml" command="load" />
	</node> -->

</launch>